mungegithub/mungers/submit-queue.go

```diff
@@ -1204,7 +1227,7 @@ func (sq *SubmitQueue) doGithubE2EAndMerge(obj *github.MungeObject) bool {
 
 	if obj.HasLabel(retestNotRequiredLabel) || obj.HasLabel(retestNotRequiredDocsOnlyLabel) {
 		atomic.AddInt32(&sq.instantMerges, 1)
-		sq.mergePullRequest(obj)
+		sq.mergePullRequest(obj, mergedSkippedRetest)
```

Comment:
eh, it was fine before when it was ignoring the error

all that will happen is that it fails the merge, removes it from the queue, and then re-adds it on the next loop